app/components/candidate_interface/application_dashboard_guidance_component.html.erb

```diff
@@ -6,17 +6,17 @@
     <p class="govuk-body">Your application has been submitted and is with the training provider.</p>
   <% end %>
 
-  <p class="govuk-body">Our <%= govuk_link_to('teacher training advisers', 'https://beta-adviser-getintoteaching.education.gov.uk/') %> can help you prepare for any interviews.</p>
+  <p class="govuk-body">Our <%= govuk_link_to('teacher training advisers', I18n.t('get_into_teaching.url_get_an_adviser')) %> can help you prepare for any interviews.</p>
```

Comment:
Super minor, but should we use the shorter `t()` helper within views? 🤔 
```suggestion
  <p class="govuk-body">Our <%= govuk_link_to('teacher training advisers', t('get_into_teaching.url_get_an_adviser')) %> can help you prepare for any interviews.</p>
```

Reply:
I just used the full `I18n.t` because it was easier and works everywhere. What's the benefit of using the shorthand? It would be nice to stick to one or the other but that's probably never going to happen 😂